portfolio/src/main/webapp/script.js

```diff
@@ -17,8 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
+      ['Hala Madrid'];
```

Comment:
Nit: This statement can now fit in one line.
`const greetings = ['Hala Madrid']`

We should keep the new line between this statement and the comment below to maintain separation between different code sections within the same function.